src/config.js

```diff
@@ -9,7 +10,7 @@ const APP_VERSION = require('../package.json').version
 
 const IS_TEST = isTest()
 const PORTABLE_PATH = IS_TEST
-  ? path.join(__dirname, '../test/tempTestData')
+  ? path.join(process.platform === 'windows' ? os.tmpdir() : '/tmp', 'WebTorrentTest')
```

Comment:
The Mac `os.tmpdir()` is different with each run of the app (you can test this!), which causes the screenshots to be different, which won't work for our integration tests